doc/source/Integration.rst

```diff
@@ -286,8 +286,7 @@ be set to their desired values by accessing ``grackle_data``.  See
 
    This structure holds all chemistry and cooling rate arrays.  All functions
    described here make use of an internally stored instance of this type.
-   The user will not normally encounter this data type, except when using the
-   :ref:`internal_functions`.
+   The user will not normally encounter this data type.
```

Comment:
How about something like the following blurb (it's a little verbose - so maybe just the first sentence?)

"The user will not encounter this datatype unless they are using the :ref:`local_functions`. In that case, users are encouraged to treat it as an opaque datatype (Functions are provided to initialize and cleanup the datatype. Additionally, the precise organization and naming of the encoded data may change between releases)."